lib/moped/sockets/ssl.rb

```diff
@@ -18,13 +20,15 @@ class SSL < OpenSSL::SSL::SSLSocket
       # @param [ Integer ] port The port.
       #
       # @since 1.2.0
-      def initialize(host, port)
+      def initialize(host, port, options)
+        @context = create_context(options)
         @host, @port = host, port
         handle_socket_errors do
           @socket = TCPSocket.new(host, port)
-          super(socket)
+          super(socket, context)
           self.sync_close = true
           connect
+          # verify_certificate!
```

Comment:
Would it be worth surrounding this with a rescue / swallow that prints something out, so we can see a behavior change when it starts working?

Reply:
it works! :) I mostly copy pasta-d it into this PR: https://github.com/crashlytics/riemann-ruby-client/pull/1 and in the description, you can see the checks I ran.